types/d3-sankey/index.d.ts

```diff
@@ -264,4 +342,4 @@ export function sankey(): SankeyLayout<{}, {}>;
  * Sankey layout generator. These properties are IN EXCESS to the properties explicitly identified in the
  * SankeyLinkMinimal interface.
  */
-export function sankey<N extends SankeyExtraProperties, L extends SankeyExtraProperties>(): SankeyLayout<N, L>;
+export function sankeyLinkHorizontal<N extends SankeyExtraProperties, L extends SankeyExtraProperties>(): Link<any, SankeyLink<N, L>, [number, number]>;
```

Comment:
@gustavderdrache What do you think in light of my above comment? Should we broaden the signatures to reflect the full scope of generics supported by `Link<...>`? I do not have a strong preference, both approaches have pros and cons and are equally workable.